core/src/test/scala/unit/kafka/common/TopicTest.scala

```diff
@@ -17,6 +17,7 @@
 
 package kafka.common
 
+import org.apache.kafka.common.errors
```

Comment:
I needed to do this because there is an InvalidTopicException in kafka.common that gets picked up unless I use errors.InvalidTopicException. If you like I can fully qualify the usage and get rid of the import all together.